tests/test_numbered_collection.py

```diff
@@ -385,3 +389,39 @@ def test_data_pop(cp_simple_problem):
     assert old_mat not in cp_simple_problem.data_inputs
     with pytest.raises(TypeError):
         cp_simple_problem.materials.pop("foo")
+
+
+# disable function scoped fixtures
+@settings(suppress_health_check=[hypothesis.HealthCheck.function_scoped_fixture])
+@given(start_num=st.integers(), step=st.integers())
+def test_mat_clone(cp_simple_problem, start_num, step):
```

Comment:
Is this not enough an integration test? This is done from a problem. `MCNP_Problem.clone` doesn't exist.

Reply:
Wow. I can't even remember what I write.